js/app.js

```diff
@@ -876,7 +876,7 @@ $(document).ready(function () {
     _utf8_decode: function (utftext) {
       let string = "";
       let i = 0;
-      let c = c1 = c2 = 0;
+      let [c, c2, c3] = Array(3).fill(0);
```

Comment:
What's the rational behind this?

Reply:
`c1` isn't used.
The new line of code declares and assigns 0 to the three respective variables which is what the original code was trying to do.

Reply:
Thank you.

My suggestion is to rename variable `c` to `c1`, so that we don't have a confusingly missing `c1` variable between `c` and `c2`.

And then use a simpler, lower-level declaration on this line:
`let [c1, c2, c3] = [0, 0, 0];`

Could you make the modifications?

Reply:
I completely agree. New branch is pushed and ready for review.

Reply:
Could you make this modification as well?
`let [c1, c2, c3] = [0, 0, 0];`

I think this is simpler, more readable, and faster by a nano second.

Reply:
Sure thing. Just pushed.